doc/source/publications.rst

```diff
@@ -9,3 +9,5 @@ Papers
 --------------
 
 - `Flower: A Friendly Federated Learning Research Framework <https://arxiv.org/abs/2007.14390>`_
+- `A first look into the carbon footprint of federated learning <https://arxiv.org/abs/2102.07627>`_
+- `On-device Federated Learning with Flower <https://arxiv.org/abs/2104.03042>`_
```

Comment:
@mariaboerner1987 please check where the Flower paper is cited and add those papers that are based on Flower (but not the ones that merely mention Flower alongside other frameworks)

Reply:
Do you have a special paper in your mind? I could only find papers citing Flower together with TFF and PySyft. 
The FjORD is missing but will be added now.

Reply:
Let's add the FjORD one for now